tests/src/Traits/ApigeeEdgeFunctionalTestTrait.php

```diff
@@ -31,26 +31,24 @@
 /**
  * Provides common functionality for the Apigee Edge test classes.
  *
- * TODO Consider renaming this trait to ApigeeEdgeFunctionalTestTrait because
- * its setup() method is clearly works only in functional tests. Re-usable
- * methods in kernel/functional tests can be remain in this trait, ex.:
- * getRandomUniqueId(), logException(), etc.
+ * TODO Move methods from this trait that can be also used in kernel tests to a
```

Comment:
Is there an issue to fix this?

Reply:
No, It is a low priority issue because it is in tests so I have created a ticket for that. I usually hunt down these kinds of issues when I need to modify tests.
Also, instead of a dedicated task for this TODO, I'd rather create a "hunt down TODOs" task because I guess there are more than one in the module.